src/components/Widgets/Widget.tsx

```diff
@@ -37,10 +37,25 @@ const refuelAllowChains: ChainId[] = [
 
 interface WidgetProps {
   starterVariant: StarterVariantType;
+  fromChain?: number;
```

Comment:
The type is very similar to the one above https://github.com/lifinance/jumper.exchange/pull/910/files#diff-c1e6733c6c5412839aca9d7ae8f74d4e0739f922a1376a092498da06b3addc8cR6 would be good to extract it

Reply:
That´s right! I tried to simplify this one as well by re-using the existing interface